ci-operator/step-registry/workers-rhel/azure-provision/workers-rhel-azure-provision-commands.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# TODO: move to image
+curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+AZURE_AUTH_LOCATION="${CLUSTER_PROFILE_DIR}/osServicePrincipal.json"
+AZURE_AUTH_CLIENT_ID="$(<"${AZURE_AUTH_LOCATION}" jq -r .clientId)"
+AZURE_AUTH_CLIENT_SECRET="$(<"${AZURE_AUTH_LOCATION}" jq -r .clientSecret)"
+AZURE_AUTH_TENANT_ID="$(<"${AZURE_AUTH_LOCATION}" jq -r .tenantId)"
+
+# log in with az
+az login --service-principal -u "${AZURE_AUTH_CLIENT_ID}" -p "${AZURE_AUTH_CLIENT_SECRET}" --tenant "${AZURE_AUTH_TENANT_ID}" --output none
+
+AZURE_REGION="$(oc get -o jsonpath='{.status.platformStatus.azure.region}' infrastructure cluster)"
+export AZURE_REGION
+
+export SSH_PRIV_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-privatekey
+export PULL_SECRET_PATH=${CLUSTER_PROFILE_DIR}/pull-secret
+export SSH_PUB_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-publickey
+
+set -x
+infra_id=$(oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster)
+
+# Get bastion RG info
+vnet_RG=$(cat ${SHARED_DIR}/resouregroup)
+echo "${vnet_RG}"
+
+# Get existing vnet info
+if [ -f "${SHARED_DIR}/customer_vnet_subnets.yaml" ]; then
+  VNET_FILE="${SHARED_DIR}/customer_vnet_subnets.yaml"
+  vnet_name=$(/tmp/yq r ${VNET_FILE} 'platform.azure.virtualNetwork')
+  computeSubnet=$(/tmp/yq r ${VNET_FILE} 'platform.azure.computeSubnet')
+fi
+
+# Get computeSubnet ID
+computeSubnetID=$(az network vnet subnet show --resource-group ${vnet_RG} --vnet-name ${vnet_name} --name ${computeSubnet} --query id -o tsv)
+
+# Start to provision rhel instances from template in existing VNET and NSG
+for count in $(seq 1 ${RHEL_WORKER_COUNT}); do
+  echo "$(date -u --rfc-3339=seconds) - Provision ${infra_id}-rhel-${count} ..."
+  # az command to configure RHEL VM's
+  az vm create --resource-group "${infra_id}-rg" \
+    --name "${infra_id}-rhel-${count}" \
+    --image "RedHat:RHEL:8-lvm-gen2:8.5.2021121504" \
```

Comment:
My last comment meant we need to keep `RHEL_IMAGE` env, but set its default value to `RedHat:RHEL:8-lvm-gen2:8.5.2021121504`